### ISSUE TITLE
Missing alt attribute in footer icon.

### ISSUE THREAD

**Issue by micmath** [bbc/a11y-tuts#1]
<img width="649" alt="screenshot 2018-01-04 11 59 52" src="https://user-images.githubusercontent.com/81645/34562577-f5f68120-f146-11e7-8f02-75c2763ac7ef.png">
